### DISCUSSION TITLE
birt output format support png？

### DISCUSSION THREAD

**Discussion by syc795** [eclipse-birt/birt#1862] [Q&A]
I know that birt supports document output in pdf, word, etc. Does birt support document output as images (e.g. jpg, png, etc.)?

**Comment by speckyspooky** [comment]
This is a very special question. BIRT doesn't create docuements for images like bmp, jpg, png.
But there are multiple problems because the most documents are multipage-documents.
Therefore the question what would be expected as output of your images?

The chart-results for example will be converted internally with Apache Batik from SVG to bmp, jpg, png - this can be configured.
From technical side it would be posible to convert PDF-documents to image files based on PDFBox (which is currently not part of BIRT) - see: [https://github.com/LibrePDF/OpenPDF/issues/346](url)

Can you little bit explain what your use case is to understand your requirements.

**Comment by hvbtup** [comment]
BIRT does not create documents for image formats.
As Thomas already wrote, first of all, BIRT can create multiple pages of output.
Converting one of the existing output formats to an image is possible outside of BIRT, but there are always restrictions.
You could try using e.g. Ghostscript to convert a PDF generated by BIRT to a bitmap image.
